lib/apiservers/engine/backends/container.go

```diff
@@ -1258,7 +1258,7 @@ func attachStreams(ctx context.Context, vc *viccontainer.VicContainer, clStdin i
 
 	// For stdin, we only have a timeout for connection.  There can be a long duration before
 	// the first entry so there is no timeout for response.
-	plClient, transport := createNewAttachClientWithTimeouts(attachConnectTimeout, 0, 0)
+	plClient, transport := createNewAttachClientWithTimeouts(attachConnectTimeout, 0, attachAttemptTimeout)
```

Comment:
The name of the cost is misleading.  This is apparently a `responseHeaderTimeout`.  Can we name it as such?  

Also the ordering of the args to `createNewAttachClientWithTimeouts` is massively counterintuitive.  Can the parameter order match the order they're being assigned to the `Transport` struct?  Connect, Header, Response seems intuitive.